stateless-asks/program/src/processor.rs

```diff
@@ -167,7 +228,128 @@ fn process_accept_offer(
             ],
         )?;
     }
-    msg!("done tx from taker to maker {}", taker_size);
+    msg!("done tx from taker to maker {}", taker_pay_size);
     msg!("done!");
     Ok(())
 }
+
+#[allow(clippy::too_many_arguments)]
+fn pay_creator_fees<'a>(
```

Comment:
Seems like this would be a really useful utility for any program using metaplex nfts!